src/v2/view-builder/views/SSOExtensionView.js

```diff
@@ -14,7 +14,17 @@ const Body = BaseForm.extend({
 
   initialize () {
     BaseForm.prototype.initialize.apply(this, arguments);
-    Util.redirectWithFormGet(this.options.currentViewState.href);
+
+    // TODO: this should be handle by the foundation,
+    // not on the view level
+    const method = this.options.appState.get('__rawResponse').remediation.value
```

Comment:
Can you rebase on my idx-js refactor PR ? We no longer have access to `__rawResponse`

Reply:
Right. This is based off master. Has your change been merged yet?

Reply:
Yes, its merged